src/ray/core_worker/transport/dependency_resolver.cc

```diff
@@ -33,7 +34,24 @@ void InlineDependencies(
       if (it != dependencies.end()) {
         RAY_CHECK(it->second);
         auto *mutable_arg = msg.mutable_args(i);
-        if (!it->second->IsInPlasmaError()) {
+        if (it->second->IsInActorError()) {
+          RAY_CHECK(task.IsActorTask());
+          RAY_LOG(INFO) << "Adding IN_ACTOR dep " << id << " to task " << task.TaskId();
+          // Add the IN_ACTOR error as the metadata value.
+          RAY_CHECK(it->second->HasMetadata());
+          const auto &metadata = it->second->GetMetadata();
```

Comment:
Metadata here is OBJECT_IN_ACTOR, and the data is RayActorObjectMetadata. This is called by driver.